docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# METRICS
+
+## Data Analysis
+
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people use this?
+
+What is the overall engagement of Snooze Tabs? **This is the standard Daily
+Active User (DAU) and Monthly Active User (MAU) analysis.** This captures data
+from the people who have the add-on installed, regardless of whether they are
+actively interacting with it.
+
+### Immediate Questions
+
+TBD
+
+### Follow-up Questions
+
+TBD
+
+## Data Collection
+
+### Server Side
+There is currently no server side component to Snooze Tabs.
+
+### Client Side
+Snooze Tabs will use Test Pilot's Telemetry wrapper with no batching of data.
+Details of when pings are sent are below, along with examples of the `payload`
+portion of a `testpilottest` telemetry ping for each scenario.
+
+* The user opens the snooze panel
+```js
+{ "event": "panel-opened" }
+```
+
+* The user snoozes a tab, choosing either a pre-defined time or a custom time
+```js
+{
+  "event": "snoozed",
+  "snooze_time": 1484345836165,
+  "snooze_time_type": "tomorrow"
+}
+```
+
+* A snoozed tab wakes up as a live tab
+```js
+{
+  "event": "woken",
+  "snooze_time": 1484345836165,
+  "snooze_time_type": "tomorrow"
+}
+```
+
+* A previously snoozed tab is focused
+```js
+{
+  "event": "focused",
+  "snooze_time": 1484345836165,
+  "snooze_time_type": "tomorrow"
+}
+```
+
+* A previously snoozed tab is closed without having been focused
+```js
+{
+  "event": "closed-unfocused",
+  "snooze_time": 1484345836165,
+  "snooze_time_type": "tomorrow"
+}
+```
+
+* A previously snoozed tab is snoozed again after waking
```

Comment:
Hmm, yeah, maybe we should be more specific that it's only tracked the first time it's resnoozed. Or, until the browser restarts - which is what the code actually does right now, because I don't persist the tracking anywhere permanent.